atlasdb-impl-shared/src/main/metrics/transaction.yml

```diff
@@ -0,0 +1,29 @@
+options:
+  javaPackage: 'com.palantir.atlasdb.transaction.impl.metrics'
+
+namespaces:
+  transaction:
```

Comment:
Should this be namespaced?

Reply:
I'd rather namespace it. Even if it might be confusing for users of our internal proxy, not namespacing it can also lead to more confusion with other internal products that we have that have their own different notion of transactions.

So better be specific here. Ideally, users of the internal proxy should know it's backed by atlas, so shouldn't be too surprised to have to look at a `atlasdb` namespaced metric.